content/blog/Intro-to-HTML-CSS-and-JavaScript/index.md

```diff
@@ -0,0 +1,79 @@
+---
+{
+    title: "Introduction to HTML, CSS, and JavaScript",
+    description: 'Introduction to the underlying concepts of HTML, CSS, and JavaScript and how they work together.',
+    published: '2019-12-16T13:45:00.284Z',
+    authors: ['MDutro'],
+    tags: ['HTML', 'CSS', 'JavaScript', 'Beginner'],
+    attached: [],
+    license: 'publicdomain-zero-1'
+}
+---
+
+
+
+
+
+# HTML, CSS, and JavaScript Concepts for Beginners
+By Micah Dutro
```

Comment:
We can remove both of these lines. They're already included on the page and including them here is duplicate

![image](https://user-images.githubusercontent.com/9100169/70942168-e90a2100-2002-11ea-90d8-1684ea3a53a9.png)